server/handlers/handlers.go

```diff
@@ -41,12 +42,25 @@ const (
 	discardedReasonInactiveUser           = "inactive_user"
 	discardedReasonDuplicatedPost         = "duplicated_post"
 	discardedReasonAlreadyAppliedChange   = "already_applied_change"
+	discardedReasonFileLimitReached       = "file_limit_reached"
+	discardedReasonEmptyFileID            = "empty_file_id"
+	discardedReasonMaxFileSizeExceeded    = "max_file_size_exceeded"
+
+	actionSourceMSTeams    = "msteams"
+	DirectMessageTrue      = "true"
+	DirectMessageFalse     = "false"
```

Comment:
```suggestion
	True      = "true"
	False     = "false"
```

Reply:
Why isn't this just a boolean, converted into `"true"` and `"false"` within the metrics layer?